packages/app/app/containers/BestNewMusicCardContainer/index.tsx

```diff
@@ -29,19 +17,20 @@ function mapStateToProps(state, ownProps) {
   };
 }
 
-function mapDispatchToProps(dispatch) {
-  return {
-    actions: bindActionCreators(
-      Object.assign(
-        {},
-        FavoritesActions,
-        ToastActions
-      ),
-      dispatch
-    )
-  };
+export interface BestNewMusicCardContainerProps extends ActionCreatorsMapObject {
+  FavoritesActions: ActionCreator<any>;
+  TostActions: ActionCreator<any>;
```

Comment:
Redux + typescript = tricky 😆 Any advice on typing here would be appreciated.

Reply:
Try `import * as FavoritesActions from '../../actions/favorites';` and then just `typeof FavoritesActions`.

Reply:
This is a lazy way of passing callbacks to components, btw. It's messy and too broad. I did it because I was lazy.

Reply:
Thanks 😄 replaced <any> with typeof for Favourites and Toast.